### PR TITLE
Issue 2146 - Modified help messages for org create & update commands

### PR DESCRIPTION
**Pull request by Anukriti2512**
Signed-off-by: Anukriti Jain <anukriti251298@gmail.com>

1) Updated the help messages for '_--max-nodes_' flag of the '_hzn exchange org create_' and '_hzn exchange org update_' commands.

Resolves issue #2146

### CODE REVIEW

File: cli/hzn.go [hunk 1]
```diff
@@ -3,10 +3,13 @@ package main
 
 import (
 	"flag"
-	"github.com/open-horizon/anax/cli/sdo"
 	"os"
 	"strings"
 
+	"github.com/open-horizon/anax/cli/sdo"
```
Comment (dabooz) [inline cli/hzn.go:9]: yeah, these imports are really in an odd order. They should be alphabetic by first letter inside the double quotes.

File: cli/hzn.go [hunk 2]
```diff
@@ -105,8 +107,8 @@ Environment Variables:
 	archCmd := app.Command("architecture", msgPrinter.Sprintf("Show the architecture of this machine (as defined by Horizon and golang)."))
 
 	exchangeCmd := app.Command("exchange", msgPrinter.Sprintf("List and manage Horizon Exchange resources."))
-	exOrg := exchangeCmd.Flag("org", msgPrinter.Sprintf("The Horizon exchange organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
-	exUserPw := exchangeCmd.Flag("user-pw", msgPrinter.Sprintf("Horizon Exchange user credentials to query and create exchange resources. If not specified, HZN_EXCHANGE_USER_AUTH will be used as a default. If you don't prepend it with the user's org, it will automatically be prepended with the -o value. As an alternative to using -o, you can set HZN_ORG_ID with the Horizon exchange organization ID")).Short('u').PlaceHolder("USER:PW").String()
+	exOrg := exchangeCmd.Flag("org", msgPrinter.Sprintf("The Horizon Exchange organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
```
Comment (dabooz) [inline cli/hzn.go:110]: I think we have to leave the case alone. Please back out those changes.

**Review by dabooz — CHANGES_REQUESTED** [review]

File: cli/hzn.go (hunk 1 above)
Comment (Anukriti2512) [reply cli/hzn.go]: Corrected!

File: cli/hzn.go (hunk 2 above)
Comment (Anukriti2512) [reply cli/hzn.go]: I have backed out all the case changes, only the create and update command changes remain

**Review by dabooz — APPROVED** [review]

**Pull request merged**